app/src/main/res/layout/item_theater.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:movie="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="theater"
+            type="woowacourse.movie.model.movie.Theater" />
+
+        <variable
+            name="onClickListener"
+            type="woowacourse.movie.ui.home.adapter.TheaterAdapter.TheaterClickListener" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:paddingHorizontal="24dp"
+        android:paddingVertical="8dp"
+        movie:theaterClickListener="@{onClickListener}"
+        movie:theaterId="@{theater.id}">
+
+        <androidx.appcompat.widget.AppCompatTextView
```

Comment:
`<androidx.appcompat.widget.AppCompatTextView />` 와 `<TextView />`는 어떤 차이가 있나요?

Reply:
결론적으로, TextView 대신 AppCompatTextView 태그를 활용하여 텍스트를 렌더링 할 필요가 없어 TextView를 활용하도록 변경하였습니다!

AppCompatTextView의 API 내부 구조를 확인해 보았을 때, 아래와 같은 사실을 확인할 수 있었습니다!

![image](https://github.com/woowacourse/android-movie-theater/assets/101035437/bd7b8e8e-d2f0-4092-9bb6-3f386fe512ff)

내용 중, 
> This will automatically be used when you use TextView in your layouts and the top-level activity / dialog is provided by appcompat. You should only need to manually use this class when writing custom views.

설명을 통하여, AppCompatActivity를 상속받은 Activity 클래스 파일에서 레이아웃을 활용할 때, TextView를 활용하면 내부 로직 상 자동으로 AppCompatTextView를 활용하게 된다는 사실을 알게 되었습니다. 따라서 다음과 같이 변경을 진행하였습니다!

```
<TextView
    android:id="@+id/theater_title_text" />
```